lib/main.js

```diff
@@ -235,10 +261,11 @@ export default {
       }
     }
 
+    const program = await this.getProgram(Linter, configurationPath);
```

Comment:
This probably needs to be in a wrapper checking whether `configurationPath` is a valid value, depending on what `Linter.findConfigurationPath` returns when it _can't_ find one. You are using this as an argument to `path.dirname` above which will throw a deprecation error if it isn't a valid string.